sys-auth/polkit/polkit-122-r1.ebuild

```
@@ -149,6 +149,8 @@ src_install() {
 			diropts -m 0700 -o polkitd
 		fi
 		keepdir /etc/polkit-1/rules.d
+		insinto /lib/systemd/system/polkit.service.d
```

Technically we should install in /usr/lib/systemd/system for merged-usr systems.

We should really add a new function to systemd.eclass to facilitate this.

In the mean time, you could do this:

```suggestion
		local unitdir="$(systemd_get_systemunitdir)"
		unitdir="${unitdir#"${EPREFIX}"}"
		insinto "${unitdir}/polkit.service.d"
```